xarray/coding/cftime_offsets.py

```diff
@@ -41,15 +41,20 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import re
+import typing
 from datetime import timedelta
 from functools import partial
-from typing import ClassVar, Optional
 
 import numpy as np
 
 from .cftimeindex import CFTimeIndex, _parse_iso8601_with_reso
+from .core.pycompat import USE_TYPING
 from .times import format_cftime_datetime
 
+if USE_TYPING:
+    if typing.TYPE_CHECKING:
```

Comment:
Can mypy interpret this if we it this on one line? e.g., `if USE_TYPING and typing.TYPE_CHECKING` ?